src/js/gi/components/Checkbox.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import _ from 'lodash';
+
+import ToolTip from '../../common/components/form-elements/ToolTip';
+
+
+/**
+ * A form checkbox with a label that can display error messages.
```

Comment:
Is it possible to use the common component for Errorable Checkbox here instead of a specific one for GIBCT?